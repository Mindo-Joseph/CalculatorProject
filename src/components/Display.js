function Display() {
  return (
    <div>
      Some stuff goes here
    </div>
  );
}
