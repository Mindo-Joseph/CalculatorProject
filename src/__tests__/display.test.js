import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Display from '../components/Display';

describe('Display', () => {
  test('component is rendered correctly', () => {
    const displayTree = renderer.create(<Display />).toJSON();
    expect(displayTree).toMatchSnapshot();
  });
});
