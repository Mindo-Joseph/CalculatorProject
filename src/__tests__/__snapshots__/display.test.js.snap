// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component is rendered correctly 1`] = `
<div
  className="display"
>
  <p
    className=""
  >
    0
  </p>
  <span
    className="displaySubtitle"
  >
     
  </span>
</div>
`;
